### ISSUE TITLE
Assignment 25

### ISSUE THREAD

**Issue by jisaacks** [tiy-tpa-js-q1-2015/assignments#231]
https://github.com/tiy-tpa-js-q1-2015/assignments/tree/master/025#031615

**Comment by tawnimartin** [comment]
Hi JD -

Here are the links to my assignment.. The one showing on ghpages is the first one I did.  But I tried another way of doing the search based on the tutorial, but that shows the value as you're typing.  I also tried one based on what we reviewed in class and could not figure it out. I think the last step is to pass the value from parent SearchBox to child SearchBar, but I can't figure out how!
https://github.com/tawnimartin/tiy-assignment-25
http://tawnimartin.github.io/tiy-assignment-25/
https://medium.com/@tawnimartin/backfire-b975d088f65d
